index.html

```diff
@@ -204,10 +204,50 @@ <h3  class="about-boxes">Skills</h3>
             </ul>
         </li> 
     </ul>
- 
-
 </section>
 
 
+
+
+
+<section class="foot-section">
+    <div class="ornament-top"><img src="/pict-mobile/image_geometry_1.svg" alt=""></div>
+<h2 class="foot-section-h">Get started</h2>
+<p class="foot-section-p">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+
+
+
+
+
+    <form action="" method="post">
+
+            <input type="text" name="your name" placeholder="Your Name" maxwidth='30' required id="">
+        
+            <input type="email" placeholder="Your email"  name="your email" id="" required >
+    
+            <textarea name="form-coment" id="" placeholder="Enter your message here" maxwidth='500' required cols="30" rows="10"></textarea>
+        
+        <button class="start-coll-btn">Start collaboration</button>
```

Comment:
- [ ] Unfortunately, your form is not working as expected because it doesn't have the `action` attribute linked with the Formspree. Kindly follow the 'get started' section in the [formspree service](https://formspree.io/html) fix the action and remember to add a `type` attribute to your 'button' element.
 - [ ] The name and message fields should have a limited amount of characters, and I see that you tried to apply the `maxwidth` property but this does not apply the character limit that you want. Kindly fix this attribute and use the correct one that limits the length of an input.
 - [ ] The form section design is almost matching with the template, but the input sizes are a little bit larger than the size expected. Kindly shrink the inputs so they can fit in the user mobile screen like the template.
 - Your design:
 - 
<img width="485" alt="image" src="https://user-images.githubusercontent.com/61919617/206041942-a22cd333-4229-4fae-a13c-81ae0d09ae51.png">

 - Template design:
 - 
<img width="301" alt="image" src="https://user-images.githubusercontent.com/61919617/206041999-ab8fb64f-bbdf-4273-8b29-491fc4fec881.png">